coderd/database/dbmem/dbmem.go

```diff
@@ -1315,6 +1316,25 @@ func (*FakeQuerier) DeleteTailnetTunnel(_ context.Context, arg database.DeleteTa
 	return database.DeleteTailnetTunnelRow{}, ErrUnimplemented
 }
 
+func (q *FakeQuerier) FavoriteWorkspace(_ context.Context, arg uuid.UUID) error {
+	err := validateDatabaseType(arg)
```

Comment:
Was this added by code gen? I thought we only add it for when `arg` is a struct (i.e. `database.*Params`). If it works, I don't see any harm though, it's more complete with the check in-place.

Reply:
I believe you're correct, it's auto-added by codegen.